vllm/executor/ray_gpu_executor.py

```diff
@@ -322,19 +213,17 @@ def _run_workers(
                                        method)(*driver_args, **driver_kwargs)
 
         # Get the results of the ray workers.
-        if self.workers:
```

Comment:
This `if` removed because `workers` should always be non-empty here (i.e. if TP > 1)

Reply:
I believe this `if` is indeed redundant. However, `workers` can indeed be empty since the ray GPU executor can be run with `TP=1`